File: Data Structures & Algorithms/sliding-window-maximum/submission-0.js
class Solution {
    /**
     * @param {number[]} nums
     * @param {number} k
     * @return {number[]}
     */
    maxSlidingWindow(nums, k) {
        
		if ( k <= 0 || k > nums.length ) return {};
		let maxone = [];
		let window = [];
		for ( let i = 0; i < k; i++ ) {
			window.push( nums[ i ] );
		}
		window.sort( ( a, b ) => b - a );
		maxone.push( window[ 0 ] );
		for ( let i = k; i < nums.length; i++ ) {
			window.splice( window.indexOf( nums[ i - k ] ), 1 )
			window.push( nums[ i ] );
			window.sort( ( a, b ) => b - a );
			maxone.push( window[ 0 ] );
		}
		return maxone;
    }
}

File: Data Structures & Algorithms/missing-number/submission-0.cs
public class Solution {
    public int MissingNumber(int[] nums) {
        int n = nums.Length;
        int total = n*(n+1)/2;
        int sum = 0;
        for(int i =0 ; i < nums.Length;i++)
        {
            sum += nums[i];
        }
        return total - sum;
    }
}